pkg/receiver/jobreceiver/README.md

```diff
@@ -0,0 +1,130 @@
+at which time the receiver will emit a single structured event capturing the
+command output and optionally metadata about the execution such as exit code
+and duration. This is useful for true monitoring jobs, such as using a
+monitoring plugin not supported natively by OTEL.
+
+When configured with the `log_entries` output type, command output will be
+emitted as distinct log entries line-by-line. This can be used to act as a
+bridge to third party logs and other log events that may not be straightforward
+to access.
+
+[stanza]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/operators/README.md
+**Output Configuration - event**
+
+| Configuration | Default | Description
+| ------------ | ------------ | ------------
+| include_command_name | true | When set, includes the attribute `command.name` in the log event.
+| include_command_status | true | When set, includes the attribute `command.status` in the log event.
+| include_command_duration | true | When set, includes the attribute `command.duration` in the log event.
+| max_body_size | | When set, restricts the length of command output to a specified [ByteSize](#bytesize-parameters).
+
+**Output Configuration - log_entries**
+
+| Configuration | Default | Description
+| ------------ | ------------ | ------------
+| include_command_name | true | When set, includes the attribute `command.name` in the log event.
+| include_stream_name | true | When set, includes the attribute `command.stream.name` in the log event. Indicating `stdin` or `stderr` as the origin of the event.
+| max_log_size | | When set, restricts the length of a log entry to a specified [ByteSize](#bytesize-parameters).
+| encoding | utf-8 | Encoding to expect from the command output. Used to detect log entry boundaries.
+| multiline | | Used to override the default newline delimited log entries.
+| multiline.line_start_pattern | | Regex pattern for the beginning of a log entry. Mutualy exclusive with end pattern.
+| multiline.line_end_pattern | | Regex pattern for the ending of a log entry. Mutualy exclusive with start pattern.
+
+### Runtime Asset Configuration
+
+//todo(ck)
+
+## Additional Features
+
+### Time Parameters
+
+Time parameters are specified as sets of decimal numbers followed by a
+unit suffix. e.g. `60s`, `45m`, `2h30m40s`.
+
+### ByteSize Parameters
+
+ByteSize parameters can be specified either as an integer number or as a string
+starting with decimal numbers optionally followed by a common byte unit
+prefixes. e.g. `16kb`, `32MiB`, `1GB`.
+
+## Example configuration
+
+```yaml
+monitoringjob:
+  schedule:
+    interval: 1h
+  exec:
+    command: check_ntp_time
+    arguments:
+        - "-H"
+        - time.nist.gov
+    timeout: 8s
+  output:
+    type: event
+    event:
+      include_command_name: false
+      include_command_status: true
+      include_command_duration: true
+      max_body_size: '32kib'
```

Comment:
Removing the stanza stuff from the receiver may simplify receiver configuration.

Reply:
At the moment I'd tend to agree. Long term, though I think we'll regret not having it.

Take a look at the Sumo Logic Script Source docs for example - there's a good bit of configuration for parsing the raw log output client side. I think we are trying to head off needing to come back to add features like this piecemeal.